config/initializers/datadog.rb

```diff
@@ -26,16 +26,12 @@
     c.tracing.instrument :redis, service_name: 'vets-api-redis'
     c.tracing.instrument :pg, service_name: 'vets-api-pg'
     c.tracing.instrument :http, service_name: 'vets-api-net-http'
-    c.tracing.log_injection.enabled = false
```

Comment:
Would this work? `c.tracing.log_injection :false` We know it's a boolean based on the error in dev:
```
NoMethodError: undefined method `enabled=' for true (NoMethodError)
    c.tracing.log_injection.enabled = false
                           ^^^^^^^^^^
/app/config/initializers/datadog.rb:29:in `block in <top (required)>'
/app/config/initializers/datadog.rb:7:in `<top (required)>'
```